src/main/java/edu/ie3/datamodel/utils/Try.java

```diff
@@ -638,7 +639,7 @@ public int hashCode() {
    */
   @FunctionalInterface
   public interface TrySupplier<T, E extends Exception> {
-    T get() throws E;
+    T get() throws E, SourceException;
```

Comment:
Why is this necessary. We would only expect an exception of type `E` here.

Reply:
Good catch. Seems to be an artefact which I didn't removed.